config/crd/bases/metal3.io_provisionings.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.2.5
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
@dhellmann This change was due to the fact that we're asking make generate to fetch v0.2.4 of controller-gen, if none was found locally. Do you want me to change this to v0.2.5 in the Makefile?

Reply:
I also think we should not look for a local copy of controller-gen in the Makefile. We should just stick to our blessed vendored version. Or else this value might get changed a lot from PR to PR. What do you think?

Reply:
Can this change be left out of this PR?

Reply:
+1. Will remove it. Thanks.